regexes/device/mobiles.yml

```diff
@@ -1732,7 +1751,7 @@ Vizio:
 
 # GOCLEVER
 GOCLEVER:
-  regex: 'GOCLEVER|QUANTUM|ARIES|INSIGNIA'
+  regex: 'GOCLEVER|QUANTUM|ARIES|INSIGNIA|ORION_|ELIPSO'
```

Comment:
Maybe better use `ORION` instead of `ORION_` as not all of those devices needs to have that `_` in the useragent.
For example: `Mozilla/5.0 (Linux; Android 4.2.2; ORION7oL Build/JDQ39) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.141 Safari/537.36`
Also as far as I had seen all `Orion` devices are tablets, so maybe you could add that separately instead of in the detection below:
```
- regex: 'ORION[ _-]?([^;/]+) Build'
  model: 'Orion $1'
  device: 'tablet'
```
You could use the useragent above to prove that.